variable/include/scipp/variable/subspan_view.h

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2020 Scipp contributors (https://github.com/scipp)
+/// @file
+/// @author Simon Heybrock
+#ifndef SCIPP_VARIABLE_SUBSPAN_VIEW_H
```

Comment:
Why not `#pragma once`?

Reply:
I only changed this in files I was editing manually. We can do the rest of the include guards another time.

Reply:
Yes, and it seemed this was a new file, so I thought you could change it here?

Reply:
For most files I just used `git mv`.